apps/src/templates/PaneHeader.story.jsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import PaneHeader, {PaneSection, PaneButton} from './PaneHeader';
+
+const styles = {
+  header: {
```

Comment:
Why do we need styles in the Story? Should they be baked into the components or are these styles already part of the page on which these components are used.

Reply:
The styles are usually baked into the page or added as need to display titles, buttons, icons, wherever requested in the spec. I included these styles to make it easier to see the multiple buttons and sections that I've added to the headers to reduce the amount of duplicating needed to show all the different combinations.